cli/flox-rust-sdk/src/models/lockfile.rs

```diff
@@ -1221,7 +1221,7 @@ impl LockedManifestCatalog {
 }
 
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
-pub struct LockedManifestPkgdb(Value);
+pub struct LockedManifestPkgdb(pub Value);
```

Comment:
nit: i think implemeting `Deref` (or deriving with `derive_more::Deref`) would be preferable to keep the inside value hidden

Reply:
Fixing